src/components/Form/CrossPodWriteForm.jsx

```diff
@@ -20,69 +19,61 @@ import FormSection from './FormSection';
  *
  * @memberof Forms
  * @name CrossPodWriteForm
+ * @returns {React.ReactElement} The form
  */
-
 const CrossPodWriteForm = () => {
   const { session } = useSession();
   const { state, dispatch } = useStatusNotification();
-  const { selectedUser } = useContext(SelectUserContext);
-  const [username, setUsername] = useState('');
   const [docType, setDocType] = useState('');
+  const [expiryDate, setExpiryDate] = useState('');
+  const [description, setDescription] = useState('');
+  const [file, setFile] = useState('');
+  const { podUrl, person } = useContext(SelectUserContext);
```

Comment:
Both `podUrl` and `person` comes out as undefined when a user is selected. Should pull data from `selectedUser` instead with `selectedUser.podUrl` and `selectedUser.person`

Reply:
fixed